cypress/locators/show-edit-pod/locators.js

```diff
@@ -1,9 +1,16 @@
 // component preview locators
-export const SHOW_EDIT_POD_CONTENT_BOX = '[data-element="content"]';
-export const SHOW_EDIT_POD_CONTENT_ELEMENTS = '[data-component="content"]';
-
-export const SHOW_EDIT_POD_TITLE = "h4[data-element=title]";
-export const SHOW_EDIT_SAVE_BUTTON = 'button[data-element="submit-button"]';
-export const SHOW_EDIT_CANCEL_BUTTON = 'button[data-element="cancel-button"]';
-export const SHOW_EDIT_DELETE_BUTTON = "[data-element=delete-button]";
-export const SHOW_EDIT_POD_EDIT = '[data-element="edit"]';
+export const SHOW_EDIT_POD = '[data-component="pod"]';
+export const CONTENT_SHOW_EDIT_POD = '[data-component="content"]';
```

Comment:
But I think there is such locator in `content`

```suggestion
export const CONTENT = '[data-component="content"]';
```